packages/passport-auth/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# @keystone-alpha/passport-auth
+
+## 1.0.0
```

Comment:
The `CHANGELOG.md` will get generated with this content by the `changeset` tool next time we bump versions, so this file should just contain:

```
# @keystone-alpha/passport-auth
```

Reply:
new to this `changeset` stuff, still learning. will fix this